psi4/driver/procrouting/proc.py

```diff
@@ -1249,12 +1249,23 @@ def scf_wavefunction_factory(name, ref_wfn, reference, **kwargs):
         # For the dimer SAPT calculation, we need to account for the external potential
         # in all of the subsystems A, B, C. So we add them all in total_external_potential
         # and set the external potential to the dimer wave function
+        from psi4.driver.qmmm import QMMMbohr
+
         total_external_potential = core.ExternalPotential()
 
         for frag in kwargs['external_potentials']:
             if frag.upper() in "ABC":
-                wfn.set_potential_variable(frag.upper(), kwargs['external_potentials'][frag].extern)
-                total_external_potential.appendCharges(kwargs['external_potentials'][frag].extern.getCharges())
+                chrgfield = QMMMbohr()
+                for qxyz in kwargs['external_potentials'][frag]:
```

Comment:
True. And it'll get worse, as nbody needs a similar function in DDD. I agree on consolidation down the road, but (1) with restructuring imminent and (2) import errors when I try such a fn in utils, I prefer to defer it.

Reply:
Makes sense to me. Because I haven't been responsible for enough growth in the issue list lately, can you create a `post-DDD` issue, similar to the `post-L2` issue? I will happily accept "this needs to wait until after the rest of DDD is in" for any code cleanup suggestion I make, so long as it gets tracked on such an issue.